dip-0004.md

```diff
@@ -98,10 +98,20 @@ A SML entry consists of the following fields:
 | platformHTTPPort | uint_16 | 0 or 2 | TCP port of Platform HTTP/API interface (network byte order). Only present for masternode type 1. |
 | platformNodeID | byte[] | 0 or 20 | Dash Platform P2P node ID, derived from P2P public key. Only present for masternode type 1. |
 
-Starting from the first block after the v19 fork, the following changes are activated:
+Clients with a protocol version greater than or equal to 70228 will receive the following field just before `proRegTxHash`.
 
-1. pubKeyOperator of all SML entries are serialised using the basic BLS scheme (otherwise, the legacy BLS scheme is used.)
-2. type field is serialised.
+| Field               | Type     | Size | Description                       |
+|---------------------| -------- | ---- |-----------------------------------|
+| nVersion            | uint16_t | 2    | Version of the SML entry          |
```

Comment:
why not make it a part of the table above with a note that it's `only for clients with a protocol version greater than or equal to 70228`?

Reply:
actually, nvm